defaults/main.yml

```diff
@@ -113,13 +113,20 @@ tor_exit_notice_file: tor-exit-notice.html
 
 # List of blacklisted DNS resolvers
 #
-# Google DNS servers taken from:
-#  https://developers.google.com/speed/public-dns/docs/using
+# Google DNS IPs:
+#  8.8.8.8
+#  8.8.4.4
+#  2001:4860:4860::8888
+#  2001:4860:4860::8844
+#  taken from: https://developers.google.com/speed/public-dns/docs/using#google_public_dns_ip_addresses
 #
 # OpenDNS IPs:
-#  208.67.222.123
-#  208.67.220.123
```

Comment:
Did you verify whether the former IPs no longer work?

Reply:
good point, they do still resolve, reverting removal